src/main/java/seedu/address/model/interview/Interview.java

```diff
@@ -90,7 +90,7 @@ public boolean isDuringOfficeHour() {
      * Returns true if this interview's date and time has expired.
      */
     public boolean isExpired() {
-        return this.interviewDateTime.isBefore(LocalDateTime.now());
+        return this.interviewEndDateTime.isBefore(LocalDateTime.now());
```

Comment:
quick minor clarification on implementation! :) -> wondering why instead of passing in `LocalDateTime.now()` at the start of the method call for `getExpiredInterviewCandidates' so that all interviews are compared against the same time stamp, the comparison is made kind of like 'dynamically' here instead?

Reply:
Wanted to make use of the abstraction we had for checking expiring interviews initially.